wiki/People/Community_Contributors/de.md

```diff
@@ -1,33 +1,220 @@
+| ![][flag_HU] [Kurokami](https://osu.ppy.sh/users/260933) | Herausragender Beitrag zum Beatmap Spotlights-Projekt |
+| ![][flag_DE] [p3n](https://osu.ppy.sh/users/123703) | Herausragender Beitrag in zahlreichen Projekten und Bereichen |
+| ![][flag_FR] [shARPII](https://osu.ppy.sh/users/776257) | Hervorragender Beitrag zum GMT und zur Aufrechterhaltung vom Wettbewerben |
+| ![][flag_US] [Toy](https://osu.ppy.sh/users/2757689) | Project Loved Teamleiter |
+| ![][flag_CA] [Kaifin](https://osu.ppy.sh/users/2596942) | Frühzeitige Unterstützung und Organisation des Project Loved |
+| ![][flag_US] [Zak](https://osu.ppy.sh/users/1375955) | Project Loved Captain (osu!catch) |
+| ![][flag_US] [Backfire](https://osu.ppy.sh/users/263110) | Project Loved captain (osu!taiko) |
+| ![][flag_DE] [Tenshichan](https://osu.ppy.sh/users/1101600) | Project Loved Captain (osu!catch) |
+| ![][flag_PL] [Kamikaze](https://osu.ppy.sh/users/2124783) | Project Loved captain (osu!mania) |
+| ![][flag_GB] [Pope Gadget](https://osu.ppy.sh/users/2288341) | Project Loved captain (osu!mania) |
+| ![][flag_AR] [Yuii-](https://osu.ppy.sh/users/2935923) | Herausragender Beitrag zum Community Mentorship Programm |
+| ![][flag_US] [Halfslashed](https://osu.ppy.sh/users/4598899) | Herausragender Beitrag zum Community Mentorship Programm |
+| ![][flag_DE] [Mir](https://osu.ppy.sh/users/8688812) | Herausragender Beitrag zum Community Mentorship Programm |
+| ![][flag_US] [Mun](https://osu.ppy.sh/users/6699165) | Herausragender Beitrag zum Community Mentorship Programm |
+| ![][flag_FI] [J1NX1337](https://osu.ppy.sh/users/3971179) | Herausragender Beitrag zum Community Mentorship Programm |
+| ![][flag_JP] [ekr](https://osu.ppy.sh/users/4497706) | Herausragender Beitrag zum osu!lazer-Projekt |
+
+## 2020
+
+### Februar
+
+*Für den Newsbeitrag, siehe: [Community Contributors: 2019](https://osu.ppy.sh/home/news/2020-02-07-community-contributors-2019)*
+
+| Benutzer | Beiträge |
+| :-- | :-- |
+| ![][flag_CA] [VINXIS](https://osu.ppy.sh/users/4323406) | Herausragende Beiträge zu Community-Angelegenheiten, Veranstaltungen und Wettbewerben |
+| ![][flag_SG] [hehe](https://osu.ppy.sh/users/2123087) | Herausragende Beiträge zur Mapping-Szene, zu Veranstaltungen und Wettbewerben |
+| ![][flag_US] [Noffy](https://osu.ppy.sh/users/1541323) | Herausragende Beiträge zur Mapping-, Modding- und Metadaten-Szene |
+| ![][flag_SG] [Shoegazer](https://osu.ppy.sh/users/2520707) | Herausragende Beiträge zum Spielmodus osu!mania |
+| ![][flag_GB] [JBHyperion](https://osu.ppy.sh/users/4879508) | Herausragende Beiträge zum Spielmodus osu!catch und zum Management |
+| ![][flag_GB] [-Mo-](https://osu.ppy.sh/users/2202163) | Herausragende Beiträge zu Management- und Führungsangelegenheiten |
+| ![][flag_US] [Chaos](https://osu.ppy.sh/users/2628870) | Herausragende Beiträge für das Global Moderation Team |
+| ![][flag_BE] [yaspo](https://osu.ppy.sh/users/4945926) | Herausragende Beiträge zum Mentorship-Projekt |
```

Comment:
Is `Mentorship project` the same as `Community Mentorship Programm`?

Reply:
Yeah, i believe it is. I'm gonna change it accordingly to your other suggestion